flink-python/pyflink/table/types.py

```diff
@@ -2068,6 +2079,12 @@ def conv(obj):
         else:
             return dict(zip(self._fields, self))
 
+    def get_row_kind(self) -> RowKind:
```

Comment:
What about changed as following?
```suggestion
    @property
    def row_kind(self) -> RowKind:
```

Reply:
The `Row` class has override the "__getattr__" method to access fields via column names. If we change to this, it will cause an ambiguity problem.